system-tests/e2e-transfer-test/runner/src/test/java/org/eclipse/edc/test/e2e/HttpProxyDataPlaneExtension.java

```diff
@@ -63,6 +66,8 @@ public void initialize(ServiceExtensionContext context) {
         var proxyUrl = "http://localhost:%d%s".formatted(portMapping.port(), portMapping.path());
         generatorService.addGeneratorFunction("HttpData", address -> Endpoint.url(proxyUrl));
 
+        generatorService.addGeneratorFunction("HttpData", () -> Endpoint.url(proxyUrl));
```

Comment:
not part of your PR, but the name of this method is confusing... should it be called `addResponseChannelGeneratorFunction`? wdyt? 
It has not a lot of usages, it could be renamed straight away.